src/erk/cli/commands/init.py

```diff
@@ -332,45 +395,47 @@ def init_cmd(
     cfg_path.write_text(content, encoding="utf-8")
     user_output(f"Wrote {cfg_path}")
 
-    # Check for .gitignore and add .env
-    gitignore_path = repo_context.root / ".gitignore"
-    if not gitignore_path.exists():
-        # Early return: no gitignore file
-        pass
-    else:
-        gitignore_content = gitignore_path.read_text(encoding="utf-8")
-
-        # Add .env
-        gitignore_content, env_added = _add_gitignore_entry_with_prompt(
-            gitignore_content,
-            ".env",
-            "Add .env to .gitignore?",
-        )
-
-        # Add .erk/scratch/
-        gitignore_content, scratch_added = _add_gitignore_entry_with_prompt(
-            gitignore_content,
-            ".erk/scratch/",
-            "Add .erk/scratch/ to .gitignore (session-specific working files)?",
-        )
-
-        # Add .impl/
-        gitignore_content, impl_added = _add_gitignore_entry_with_prompt(
-            gitignore_content,
-            ".impl/",
-            "Add .impl/ to .gitignore (temporary implementation plans)?",
-        )
-
-        # Write if any entry was modified
-        if env_added or scratch_added or impl_added:
-            gitignore_path.write_text(gitignore_content, encoding="utf-8")
-            user_output(f"Updated {gitignore_path}")
-
-    # Offer to add erk permission to repo's Claude Code settings
-    offer_claude_permission_setup(repo_context.root)
+    # Interactive prompts (skip if --no-interactive)
+    if not no_interactive:
```

Comment:
avoid double negative  
  
  
also let's try to avoid adding this much indentation